command/push.py

```diff
@@ -1,8 +1,35 @@
+import base64
+import os
+import requests
+import tarfile
+
 class PushCmd():
     name = 'push'
 
     def __init__(self):
         pass
         
-    def push(self):
-        print("Hello, world. The push feature is currently under construction.")
+    def push(self, bundle_dir, namespace, repository, release, auth_token):
+        base64_bundle = self.__create_base64_bundle(bundle_dir, repository)
+        self.__push_to_registry(namespace, repository, release, base64_bundle, auth_token)
+
+    def __create_base64_bundle(self, bundle_dir, repository):
```

Comment:
There is a planned usecase to use this library without providing files. We should:
1. rewrite this method to take a `String`.
2. Add a method that takes a file and provides it's content to the method mentioned in step 1.

OR

Create an issue to track the string usecase.

Reply:
Understood. LGTM. :+1: